web/scripts/app.js

```diff
@@ -1960,6 +1960,7 @@ export class ComfyApp {
 		for (const outerNode of this.graph.computeExecutionOrder(false)) {
 			const skipNode = outerNode.mode === 2 || outerNode.mode === 4;
 			const innerNodes = (!skipNode && outerNode.getInnerNodes) ? outerNode.getInnerNodes() : [outerNode];
+			const groupTitle = outerNode.getInnerNodes ? outerNode.title : undefined;
```

Comment:
This is janky, but there doesn't seem to be any metadata on group nodes that indicates they're group nodes, and this is how Comfy itself tells that a node is a group.